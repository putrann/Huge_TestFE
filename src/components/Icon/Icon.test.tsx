import * as React from 'react';
import { render } from '@testing-library/react';
import { Icon } from '.';

describe('<Icon/>', () => {
  it('Should match snapshot', () => {
    const { container } = render(<Icon type="search" />);
    expect(container.firstChild).toMatchSnapshot();
  });

  it('Should not render anything if type mismatch or undefined', () => {
    // @ts-ignore
    const { container } = render(<Icon />);
    expect(container.firstChild).toBeNull();
  });
});
