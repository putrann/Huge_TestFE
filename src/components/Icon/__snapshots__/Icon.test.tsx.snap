// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Icon/> Should match snapshot 1`] = `
<svg
  fill="currentColor"
  height="1em"
  width="1em"
>
  search.svg
</svg>
`;
